new-system-tests/src/test/resources/apicurio/01_subscription_namespace.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
We test two operator installation types:
- cluster-wide (operator is able to operate in any namespace)
- "namespaced" (operator should be able to operate in specified (target) namespace(s) only)

This is defined in `spec.targetNamespaces` of `OperatorGroup` in namespace of operator installation.

In Kubernetes is pre-prepared namespace `operators`, in OpenShift is pre-prepared namespace `openshift-operators`, both for cluster-wide installations.

This is (probably) not possible to achieve with Kubernetes Client JUnit extension? Because I need to know namespace name to use it in `spec.targetNamespaces` at the moment when creating `OperatorGroup` (it is not possible when `OperatorGroup` is defined statically in YAML file and loaded by `@LoadKubernetesManifests`).

I will appreciate any idea/help to achieve "namespaced" installation of operator.

Reply:
The "namespaced" installation "should" default to the same namespace where the operator is installed (at least in Java Operator SDK), need to check with @jsenko what the default is in this case, so, probably, simply deploying in the target namespace already covers the use case.

That said, maybe, we can verify if this operator configuration test is better suited for the integration tests in the operator repository, wdyt?